_topic_maps/_topic_map.yml

```diff
@@ -3655,7 +3655,7 @@ Topics:
     File: ossm-about
   - Name: Service Mesh 2.x release notes
     File: servicemesh-release-notes
-  - Name: Service Mesh architecture
+  - Name: Understanding Service Mesh
```

Comment:
```suggestion
  - Name: Understanding Service Mesh
```

Reply:
This adds the title "Understanding Service Mesh" twice: One at the  "top level" menu item and one immediately underneath. The adjacent repetition looks a little odd. I'd suggest the 2nd level title be "A network of microservices", but honestly, I am not at all sure that's an improvement.